src/popup/App.vue

```diff
@@ -18,12 +18,16 @@
     >
       <Header v-if="showStatusAndHeader" />
 
-      <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
-        <RouterView
-          :class="{ 'show-header': showStatusAndHeader }"
-          class="main"
-        />
-      </transition>
+      <div :class="['app-container', {scrollable: $route.meta.scrollable}]">
+        <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
+          <Scrollable :hide-scrollbar="!$route.meta.scrollable">
```

Comment:
The "Scrollable" component should not be used to wrap the whole page in order to allow to scroll it. The whole `app-inner` should scroll as in the future the header will be semi-transparent and user should see the content under the header.

Reply:
if we keep `app-inner` scrollable the scrollbar will be cut off by the header see [Liubov Comments](https://aeternity-blockchain.atlassian.net/browse/SW-342?focusedCommentId=11514)